### ISSUE TITLE
File not found exception on Xubuntu 20.04

### ISSUE THREAD

**Issue by jawuku** [oakes/Paravim#8]
Hello, could you please look into a problem I have, trying to run Paravim on Xubuntu 20.04.
When I try to launch Paravim with the command
```sh
clojure -Sdeps "{:deps {paravim {:mvn/version \""RELEASE\""}}}" -m paravim.start
```
It gives the error
```
Execution error (FileNotFoundException) at java.io.FileInputStream/open0 (FileInputStream.java:-2).
-Sdeps (No such file or directory)

Full report at:
/tmp/clojure-9275061162753256078.edn
```
Listing the file at /tmp/clojure-9275061162753256078.edn
```
{:clojure.main/message
 "Execution error (FileNotFoundException) at java.io.FileInputStream/open0 (FileInputStream.java:-2).\n-Sdeps (No such file or directory)\n",
 :clojure.main/triage
 {:clojure.error/class java.io.FileNotFoundException,
  :clojure.error/line -2,
  :clojure.error/cause "-Sdeps (No such file or directory)",
  :clojure.error/symbol java.io.FileInputStream/open0,
  :clojure.error/source "FileInputStream.java",
  :clojure.error/phase :execution},
 :clojure.main/trace
 {:via
  [{:type java.io.FileNotFoundException,
    :message "-Sdeps (No such file or directory)",
    :at [java.io.FileInputStream open0 "FileInputStream.java" -2]}],
  :trace
  [[java.io.FileInputStream open0 "FileInputStream.java" -2]
   [java.io.FileInputStream open "FileInputStream.java" 219]
   [java.io.FileInputStream <init> "FileInputStream.java" 157]
   [java.io.FileInputStream <init> "FileInputStream.java" 112]
   [clojure.lang.Compiler loadFile "Compiler.java" 7571]
   [clojure.main$load_script invokeStatic "main.clj" 475]
   [clojure.main$script_opt invokeStatic "main.clj" 535]
   [clojure.main$script_opt invoke "main.clj" 530]
   [clojure.main$main invokeStatic "main.clj" 664]
   [clojure.main$main doInvoke "main.clj" 616]
   [clojure.lang.RestFn applyTo "RestFn.java" 137]
   [clojure.lang.Var applyTo "Var.java" 705]
   [clojure.main main "main.java" 40]],
  :cause "-Sdeps (No such file or directory)"}}
```
Here are some details of my system. I am running a 64-bit AMD processor.
```sh
lsb_release -a

Distributor ID:	Ubuntu
Description:	Ubuntu 20.04 LTS
Release:	20.04
Codename:	focal

java --version

openjdk 11.0.7 2020-04-14
OpenJDK Runtime Environment (build 11.0.7+10-post-Ubuntu-3ubuntu1)
OpenJDK 64-Bit Server VM (build 11.0.7+10-post-Ubuntu-3ubuntu1, mixed mode, sharing)

Clojure 1.10.1
```
I would be very grateful for your help.
Thank you

**Comment by oakes** [comment]
Interesting, I haven't seen this before. Can you try running the command with a few double quotes removed? I wonder if they're confusing your terminal:

```
clojure -Sdeps "{:deps {paravim {:mvn/version \"RELEASE\"}}}" -m paravim.start
```

**Comment by jawuku** [comment]
Thanks for getting back to me. I think I have found the problem.
It seems there is a bug in the Ubuntu and Debian repository packaging of Clojure; some dependencies are omitted.
 
https://bugs.launchpad.net/ubuntu/+source/clojure/+bug/1872595
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=902763

I have a habit of installing most things via the package manager, rather than manually,  so I had gone down that route instead.

I simply uninstalled that package, and downloaded the Clojure software off the official site.

Paravim now works as advertised, using the original start up command. Sorry for the oversight.

Thanks
